lib/cli/src/extract.ts

```diff
@@ -12,7 +12,7 @@ const read = async (url: string) => {
   await page.goto(url);
 
   await page.waitForFunction(
-    'window.__STORYBOOK_STORY_STORE__ && window.__STORYBOOK_STORY_STORE__.extract && window.__STORYBOOK_STORY_STORE__.extract()'
+    'window.__STORYBOOK_PREVIEW__ && window.__STORYBOOK_PREVIEW__.extract && window.__STORYBOOK_PREVIEW__.extract()'
```

Comment:
@tmeasday we should fall back to `__STORYBOOK_STORY_STORE__` if `window.__STORYBOOK_PREVIEW__.extract` doesn't exist so that if user runs `npx sb extract` on an older version of SB, it doesn't error.

Reply:
But aren't they by definition using the right version of SB?

Reply:
No. `extract` runs puppeteer against a running storybook. That storybook could be **any** version AFAIK.

Meanwhile we encourage people to run `npx sb` so that they get the latest version of the CLI. So as soon as we release this as stable, `npx sb` will assume that the Storybook it's running against has the changes from this PR (unless the user has installed a specific version of the `sb` or `@storybook/cli` package, which we discourage). And that wouldn't be the case unless they've also upgraded their Storybook, which is a completely separate operation.

Reply:
Ahh. OK. Will fix.

Reply:
@shilman perhaps what we could do is add a test for `sb extract` targeting a bunch of chromatic storybook URLs, of various versions, this way we can assert we stay compatible?

Reply:
Sounds like a good idea @ndelangen. Would be good to do the same for Chromatic's version of extract also TBH. I'll get the set of uploaded SBs together.

Reply:
I've updated the code for now and will let somebody follow up with extra tests